README.md

```diff
@@ -1,190 +1,145 @@
-path when it indicates that the code portion is used for test or example. A pre-trained Path Model [is available here](https://github.com/SAP/credential-digger/releases/download/PM-v1.0.1/path_model-1.0.1.tar.gz).
+- Path Model: Identify the test files, documentation, or example files containing fake credentials (e.g, unit tests)
+- Snippet Model: Identify the portion of code used to authenticate with passwords, and distinguish between real and dummy passwords.
+
+Download the binaries :
 
-After installing _credentialdigger_, our pre-trained Path Model can be installed as follows.
 ```bash
 export path_model=https://github.com/SAP/credential-digger/releases/download/PM-v1.0.1/path_model-1.0.1.tar.gz
+export snippet_model=https://github.com/SAP/credential-digger/releases/download/SM-v1.0.0/snippet_model-1.0.0.tar.gz
+```
+```bash
 python -m credentialdigger download path_model
+python -m credentialdigger download snippet_model
 ```
 
-### Code Snippet Model
-
-The code Snippet model identifies the password based authentication in a code and differeciate between real and fake passwords. A pre-trained Code Snippet Model [is available here](https://github.com/SAP/credential-digger/releases/download/SM-v1.0.0/snippet_model-1.0.0.tar.gz)
+> **WARNING**: If you build the code from scratch (i.e., you don't install the client via
+pip), don't run the download command from the installation folder of
+_credentialdigger_ in order to avoid errors in linking.
 
-WARNING: This Model is pre-trained with synthetic data in order to protect privacy. It helps to reduce the False Positives related to password recongnition, but with a lower precision compared to a Model pre-trained with real data.
+> **WARNING**: We provide the pre-trained models, but we do not guarantee the efficiency of these models. If you want more accuracte machine learning models, you can train your own models (just replace the binaries by your own models) or use the fine-tuning option.
 
-### Integrate ML Models in the UI
 
-**ML models are not supported yet in the UI** "out of the box", but they can be
-enabled with a little effort.
+## Usage
 
-To use ML models in the docker container running the ui, set their address in
-the `.env` file (the pre-trained Path and Snippet Models are already set).
-If the address is correct, the model will be installed at container building
-time.
-However, the current server is not executing any model during the scan of a
-repository. To enable a model, it must be listed as an argument in the `scan`
-function (as would be done in the client) in the `ui/server.py` file.
+To instanciate a client, connected to the database :
 
+```python
+from credentialdigger.cli import Client
+c = Client(dbname='MYDB', dbuser='POSTGRES_USER',
+						  dbpassword='POSTGRES_PASSWORD',
+						  dbhost='localhost', dbport=5432)
+```
+### Scan a repository
 
-## Usage (client)
+You first need to add the repository to the database. Then you can proceed to a scan :
```

Comment:
`scan` already adds the repository to the database

On the contrary, what is needed before a scan is adding rules
```
c.add_rules_from_file(...)  # To be done once (before the first scan)
c.scan(...)
```

Reply:
Ok, I'll correct it, I added a section "Configure the regular expressions Scanner" in the installation process